app/uk/gov/hmrc/tai/viewModels/benefit/RemoveCompanyBenefitsCheckYourAnswersViewModel.scala

```diff
@@ -39,7 +45,7 @@ case class RemoveCompanyBenefitsCheckYourAnswersViewModel(
 
     val stopDateLine = CheckYourAnswersConfirmationLine(
       Messages("tai.checkYourAnswers.dateBenefitEnded"),
-      stopDate,
+      stopDate.format(DateTimeFormatter.ofPattern("d MMMM yyyy")), //TODO add welsh?
```

Comment:
Why would a date need to added in Welsh?

Reply:
Got spot, currently it would print as 1 April 2023 rather than 1 Ebrill 2023

Reply:
`Dates.formatDate(LocalDate.parse(stopDate.toString()))`
is what you need

Reply:
No because we need a language which we don't have here. I have the code I'll push it later once we've demoed this